frontend/src/lib/components/TaxonomicFilter/infiniteListLogic.ts

```diff
@@ -77,14 +83,15 @@ export const infiniteListLogic = kea<infiniteListLogicType>({
         expand: true,
     },
 
-    reducers: {
+    reducers: ({ props }) => ({
         index: [
-            0 as number,
+            (props.popperEnabled === false ? -1 : 0) as number,
```

Comment:
Actually... this doesn't make sense: "If you disable the popup we unselect the first item.". This is a side-effect of `popperEnabled` that's really not obvious. Would a separate prop like `selectFirstItem = false` make more sense?